config/action.d/pf.conf

```diff
@@ -104,6 +139,10 @@ block = block quick
 # Notes.:  internally used by config reader for interpolations.
 # Values:  [ tcp | udp | icmp | ipv6-icmp ] Default: tcp
 #
+# If set to "{ tcp, udp, icmp, ipv6-icmp }", all protocols will be
+# taken care of. Similarly combination of protocols may be handled
+# in a similar manner.
+#
```

Comment:
Remove the "If." 

What is meant by "taken care of?"

How about: Multiple protocols can be specified by enclosing them in braces. For example "{ tcp, udp, icmp, ipv6-icmp }" will configure the the reader for all protocols.

Reply:
Aye. Was actually meaning to come back to this. Going to remove the braces bit entirely.

The braces should be moved to the init bit, so they are not needed and the user can just give a comma delimited list.